docker-compose/docker-compose.yaml

```diff
@@ -559,7 +559,7 @@ services:
       test: '/liveness.sh'
       interval: 10s
       timeout: 1s
-      retries: 10
+      retries: 360
```

Comment:
If the health of a container can fail for a full hour, is this healthcheck really even doing anything valuable? Perhaps modifying the start_period is a better option here, if this is intended to fix the same slow startup issue as in https://github.com/sourcegraph/deploy-sourcegraph/pull/4136.

https://docs.docker.com/engine/reference/builder/#healthcheck
> start period provides initialization time for containers that need time to bootstrap. Probe failure during that period will not be counted towards the maximum number of retries. However, if a health check succeeds during the start period, the container is considered started and all consecutive failures will be counted towards the maximum number of retries.